common/Helpers/RuntimeHelper.cs

```diff
@@ -2,22 +2,24 @@
 // Licensed under the MIT license.
 
 using System.Runtime.InteropServices;
-using System.Text;
 
 namespace DevHome.Common.Helpers;
 
-public class RuntimeHelper
+public static class RuntimeHelper
 {
-    [DllImport("kernel32.dll", CharSet = CharSet.Unicode, SetLastError = true)]
-    private static extern int GetCurrentPackageFullName(ref int packageFullNameLength, StringBuilder? packageFullName);
-
     public static bool IsMSIX
     {
         get
         {
-            var length = 0;
+            const int APPMODEL_ERROR_NO_PACKAGE = 15700;
+
+            uint length = 0;
+
+            return GetCurrentPackageFullName(ref length, null) != APPMODEL_ERROR_NO_PACKAGE;
 
-            return GetCurrentPackageFullName(ref length, null) != 15700L;
+            // See: https://learn.microsoft.com/windows/win32/api/appmodel/nf-appmodel-getcurrentpackagefullname
+            [DllImport("kernel32", ExactSpelling = true, CharSet = CharSet.Unicode)]

```

Comment:
```suggestion
            [DllImport("kernel32.dll", ExactSpelling = true, CharSet = CharSet.Unicode)]

```

I would recommend keeping the explicit suffix to avoid the suffix work and avoid encouraging other devs to do this and run into edge cases (e.g., [filenames containing periods](https://learn.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-loadlibraryexw#parameters), xplat scenarios, etc.).

Reply:
Fixed as part of switching this to use CsWin32, which also generates all P/Invoke-s with the .dll suffix 🙂